### ISSUE TITLE
Respect robots.txt

### ISSUE THREAD

**Issue by stijn-uva** [medialab/hyphe#376]
If I recall correctly, Hyphe currently does not respect robots.txt files. Depending on their ethical framework it may however be important for researchers to be able to exclude pages that are explicitly marked as non-crawlable.

Could this be made an option? I don't know exactly how Hyphe interfaces with Scrapy, but perhaps Scrapy's [robots.txt middleware](https://docs.scrapy.org/en/latest/topics/downloader-middleware.html#module-scrapy.downloadermiddlewares.robotstxt) could be toggled for this?

A nice bonus would be if it were possible to also exclude links marked with `rel="nofollow"`, but I see this is already covered in another issue (#86).

**Comment by boogheta** [comment]
Yes that definitely would be a nice feature, and not too complex to implement among other advanced crawl features at least via the API for a start
